src/main/kotlin/nl/info/zac/admin/ZaakafhandelParameterBeheerService.kt

```diff
@@ -380,9 +379,11 @@ class ZaakafhandelParameterBeheerService @Inject constructor(
     private fun mapBetrokkeneKoppelingen(
```

Comment:
This could probably do with a unit test? Or maybe an adjustment of the integration test? Not sure.